### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by TheHamkerCat**

### CODE REVIEW

File: wbb/modules/admin.py [hunk 1]
```diff
@@ -342,3 +345,44 @@ async def unpin(_, message: Message):
...
+            victim = username
+        else:
+            victim = message.reply_to_message.from_user.id
+        await message.chat.restrict_member(victim,
+                                           permissions=ChatPermissions())
+        await message.reply_text("Muted!")
+    else:
+        await message.reply_text("Get Yourself An Admin Tag!")
+
+# Unmute members
```
Comment (dashezup) [inline wbb/modules/admin.py:370]: I don't think you need comment for this. but if you want to, use function docstrings instead.

File: wbb/modules/misc.py [hunk 2]
```diff
@@ -169,3 +174,53 @@ async def ipaddrinfo(_, message: Message):
     else:
         await message.reply_text('`/ipinfo` Requires An IP As Argument'
                                  + '. Ex - `/ipinfo 1.1.1.1`')
+
+# Cheat.sh
+
+
+@app.on_message(cust_filter.command(commands=("cheat")) & ~filters.edited)
+async def cheat(_, message: Message):
+    text = message.text.replace("/cheat ", "")
```
Comment (dashezup) [inline wbb/modules/misc.py:183]: also here, use function docstrings instead

File: wbb/modules/admin.py (hunk 1 above)
Comment (TheHamkerCat) [reply wbb/modules/admin.py:370]: i used them everywhere

File: wbb/modules/misc.py (hunk 2 above)
Comment (TheHamkerCat) [reply wbb/modules/misc.py:183]: where?

Comment (dashezup) [reply wbb/modules/misc.py:183]: `# Cheat.sh`
this may be moved into function docstrings

**Pull request merged**